config/secrets.yml

```
@@ -52,6 +52,7 @@ default: &default
     api_key: <%= ENV["ETHERPAD_API_KEY"] %>
     api_version: "1.2.1"
   uclg:
+    page_with_background: <%= ENV["PAGE_WITH_BACKGROUND"] %>
     video_url: <%= ENV["VIDEO_URL"] %>
```

```suggestion
  uclg:
    conferences:
      - conference_id: 999
        program_background: /url/no-need-for-secrecy
        presentation_video: https://yotuyyo
```
Maybe a format like this would be more scalable for future conferences